Sources/Extensions/String+Array.swift

```diff
@@ -44,6 +44,25 @@ extension String {
     }
 }
 
+extension NSMutableAttributedString {
```

Comment:
Suggestion: ``` swift 
extension NSMutableAttributedString {
    func append(value: String, font: UIFont, color: UIColor) {
        append(value: value,attributes:  [
                .font: font,
                .foregroundColor: color
            ])
    }

    func append(value: String, color: UIColor) {
        append(value: value,attributes: [.foregroundColor: color] )
    }

    func append(value: String, attributes: [NSAttributedString.Key: Any]?) {
        append(
            NSAttributedString(string: value,
                               attributes: attributes))
    }
}
```

Reply:
What's the purpose of having this? ``` func append(value: String, attributes: [NSAttributedString.Key: Any]?) {
        append(
            NSAttributedString(string: value,
                               attributes: attributes))
    }``` it's just a wrapper and the logic is exactly the same as the ```append``` method swift provides us.